easybuild/easyconfigs/r/RSEM/RSEM-1.3.3-foss-2021b.eb

```diff
@@ -0,0 +1,52 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	GPLv3.0
+#
+# Notes::
+# Updated to foss-2021b
+# J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'RSEM'
+version = '1.3.3'
+
+homepage = 'https://deweylab.github.io/RSEM/'
+description = "RNA-Seq by Expectation-Maximization"
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'openmp': True, 'opt': True}
+
+source_urls = ['https://github.com/deweylab/RSEM/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['RSEM-1.3.0_makefiles.patch']
+checksums = [
+    '90e784dd9df8346caa2a7e3ad2ad07649608a51df1c69bfb6e16f45e611a40dc',  # v1.3.3.tar.gz
+    '2d244659206c78655b92f1bd519ee65f28a6b5f9418dfad04e887b64eca6641b',  # RSEM-1.3.0_makefiles.patch
+]
+
+skipsteps = ['configure']
+
+installopts = "prefix=%(installdir)s"
+
+dependencies = [
+    ('ncurses', '6.2'),
+    ('zlib', '1.2.11'),
+    ('Perl', '5.34.0'),
+    ('R', '4.1.2'),
+    ('HISAT2', '2.2.1'),
```

Comment:
These two dependencies cannot be mixed
* R/4.1.2-foss-2021b depends on HDF5/1.12.1
* HISAT2/2.2.1-gompi-2021b depends on ncbi-vdb/2.11.2-gompi-2021b, which depends on HDF5/1.10.8-gompi-2021b

Since ncbi-vdb/2.11.2-gompi-2021b is limited to HDF5 v1.10, the only solution to have RSEM in foss/2021b would be to make different easyconfig of R v4.1.2 using HDF5 v1.10 as well (not sure if it is possible). Maybe it's not worth the effort, it will be easier to move RSEM to an older toolchain using HDF5 v1.10 by default.

Reply:
@boegel it's very weird how the tests in generoso came to pass, it should not even be possible to load the modules of the dependencies.